wildfly/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -1592,7 +1592,9 @@ private void parseMemoryOffHeap(XMLExtendedStreamReader reader, ModelNode cache,
                     break;
                 }
                 case ADDRESS_COUNT: {
-                    MemoryOffHeapConfigurationResource.ADDRESS_COUNT.parseAndSetParameter(value, offHeap, reader);
+                    if (namespace.since(11, 0)) {
+                        throw ParseUtils.unexpectedElement(reader);
+                    }
```

Comment:
We should warn that the attribute has been ignored via `ignoreAttribute`.

Reply:
Sure